ui/components/app/detected-token/detected-token-ignored-popover/detected-token-ignored-popover.js

```diff
@@ -17,50 +29,94 @@ const DetectedTokenIgnoredPopover = ({
 
   const footer = (
     <>
-      <Button
-        className="detected-token-ignored-popover__ignore-button"
-        type="secondary"
-        onClick={onCancelIgnore}
+      <Box
+        display={Display.Flex}
+        justifyContent={JustifyContent.center}
+        marginLeft={4}
+        marginRight={4}
       >
-        {t('cancel')}
-      </Button>
-      <Button
-        className="detected-token-ignored-popover__import-button"
-        type="primary"
-        onClick={handleClearTokensSelection}
-      >
-        {t('confirm')}
-      </Button>
+        <Button
+          className="detected-token-ignored-popover__ignore-button"
+          block
+          variant={BUTTON_VARIANT.SECONDARY}
```

Comment:
- hover problem

Currently, when hovering over the 'cancel' button, the background color changes to blue, which is the default behavior for secondary buttons. However, the desired outcome is for the background color to remain consistent even after hovering. Please refer to the provided image after hovering for a visual representation of the issue.

I'm attempting to avoid using secondary buttons and am instead applying a hover style using SCSS. However, for some reason, the style is not updating as expected.

### cancel button was hover in image 

![after hover](https://github.com/MetaMask/metamask-extension/assets/81921291/b12a7d4a-da40-4583-b820-1830e738d97f)

Reply:
Got it